breathecode/registry/views.py

```diff
@@ -1801,7 +1801,17 @@ def get(self, request, cluster_slug=None, academy_id=None):
         if like is not None and like != "undefined" and like != "":
             items = items.filter(Q(slug__icontains=slugify(like)) | Q(title__icontains=like))
 
-        items = items.filter(**lookup)
+        lang = request.GET.get("lang", None)
```

Comment:
It should look like this one: https://github.com/breatheco-de/apiv2/blob/21d6cfe26a242526360c91ae1b827346348e7c50/breathecode/registry/views.py#L212